File: URI-py/1047.py
# -*- coding: utf-8 -*-
hi, mi, hf, mf = map(int,input().split())
hdd = hi-hf
if (hi==hf) and (mi==mf):
    print("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
elif hdd==(-1) and mi > mf:
    md = (60-mi)+mf
    print("O JOGO DUROU 0 HORA(S) E %d MINUTO(S)" %(abs(md)))
elif hi < hf:
    if mi > mf:
        md = (60-mi)+mf
    elif mi < mf:
        md = mf - mi
    elif mi == mf:
        md = mf - mi
    print("O JOGO DUROU %d HORA(S) E %d MINUTO(S)" %(hf-hi,abs(md)))
elif hi==hf and mi>mf:
    md = (60-mi)+mf
    print("O JOGO DUROU 23 HORA(S) E %d MINUTO(S)" % (abs(md)))
elif hi==hf and mf>mi:
    md = mf-mi
    print("O JOGO DUROU 0 HORA(S) E %d MINUTO(S)" % (abs(md)))
'''
elif hi > hf:
    if mi > mf:
        md = mf-mi
    elif mi < mf:
        md = mf-mi
    elif mi == mf:
        md = mf-mi
    print("O JOGO DUROU %d HORA(S) E %d MINUTO(S)33" %(abs(hf-hi),abs(md)))
'''
'''
hi, mi, hf, mf = map(int,input().split())
hdd = hi-hf
if (hi==hf) and (mi==mf):
    print("O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)")
elif hdd==(-1) and mi > mf:
    md = (60-mi)+mf
    print("O JOGO DUROU 0 HORA(S) E %d MINUTO(S)22" %(abs(md)))
elif hi < hf:
    if mi > mf:
        md = (60-mi)+mf
    elif mi < mf:
        md = mf - mi
    elif mi == mf:
        md = mf - mi
    print("O JOGO DUROU %d HORA(S) E %d MINUTO(S)44" %(hf-hi,abs(md)))
elif hi==hf and mi>mf:
    md = (60-mi)+mf
    print("O JOGO DUROU 0 HORA(S) E %d MINUTO(S)55" % (abs(md)))
elif hi==hf and mf>mi:
    md = mf-mi
    print("O JOGO DUROU 0 HORA(S) E %d MINUTO(S)66" % (abs(md)))
'''

File: URI-cpp/1047.cpp
#include <iostream>
#include <iomanip>
#include <vector>
#include <algorithm>

using namespace std;

int main()
{
    int hi,mi,hf,mf,horas,minutos,a = 0;

    cin >> hi >> mi >> hf >> mf;

    horas = hf - hi;
    minutos = mf - mi;

    if (horas < 0) {
        horas = 24 + horas;
    }
    if (minutos < 0) {
        minutos = 60 + minutos;
        horas--;
        if (horas < 0) {
            horas = 23;
        }
    }
    if (hi == hf && mi == mf) {
        cout << "O JOGO DUROU 24 HORA(S) E 0 MINUTO(S)" << endl;
    } else {
        cout << "O JOGO DUROU " << horas << " HORA(S) E " << minutos << " MINUTO(S)" << endl;
    }

    return 0;
}
